.github/workflows/deploy-to-control-plane-review-app.yml

```
@@ -351,6 +371,12 @@ jobs:
           token: ${{ secrets.CPLN_TOKEN_STAGING }}
           org: ${{ vars.CPLN_ORG_STAGING }}
 
+      - name: Print Comment id as job output
+        run: echo '${{ needs.process-deployment.outputs.comment_id }}'
```

actionlint (1.7.4)

property "process-deployment" is not defined in object type {build: {outputs: {image_tag: string}; result: string}}

(expression)